.github/CODEOWNERS

```diff
@@ -96,7 +96,6 @@
 /pkg/kubestatemetrics                    @DataDog/container-integrations
 /pkg/security/                          @DataDog/agent-security
 
-/releasenotes/                          @DataDog/agent-all
```

Comment:
The problem with this is that `@DataDog/agent-platform` will have to approve all PRs with a release note now (see #4456 that introduced this line). The two possible ways out of this are
1.  assigning ownership to someone/some team that does not have write access to this repository (so maybe @/DataDog/dev or @/noone) or
2. Remove https://github.com/DataDog/datadog-agent/blob/a5323b70dbbe13aeaae8e8702a98911b8649b03b/.github/CODEOWNERS#L12 and assign to Agent Platform all the individual files it should own.

Reply:
> assigning ownership to someone/some team that does not have write access to this repository (so maybe @/DataDog/dev or @/noone

Not sure I get this part.

> and assign to Agent Platform all the individual files it should own.

How would I know which files need to be owned by agent-platform?

Reply:
>>  assigning ownership to someone/some team that does not have write access to this repository (so maybe @/DataDog/dev or @/noone
>
> Not sure I get this part.

As stated in [the docs](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners)  *The people you choose as code owners must have write permissions for the repository.* If you assign it to someone without write access it will silently fail (which is what was happening until now).

> How would I know which files need to be owned by agent-platform?

Every file that is not owned by another team is owned by Agent Platform.

Reply:
I think solution 2 is the way to go, if it allows us to remove that line. I understand the motivation behind the initial PR, but I also think the way it currently is isn't optimal because having `agent-all` own `/releasenotes` basically puts `agent-all` on every PR, causing people to get emails for every PR just due to release notes.

I think removing it also wouldn't be bad, because usually release notes are accompanied by changes to other code, which then would notify the necessary owners anyways.